accepted/004-manifests.md

```diff
@@ -0,0 +1,54 @@
+# Manifests
+
+One of the main gripes about Kubernetes, is the management of YAML and the need to making the YAML turing complete with templates, logic, wrappers, and management utilities.
+
+The YAML merely represents the specific values of an instance of a struct in Go.
+
+We can easily adopt the "define your application in a manifest" paradigm, without actually using any of the native aurae functions to mutate a system.
+
+### Simple Manifest
+
+This manifest can be sourced from higher level applications, or even other aurae scripts.
+Application teams will be able to define their applications directly as valid aurae code.
+
+Notice how the application can be defined without connecting to an Aurae system, or accessing any of the subsystems at runtime.
```

Comment:
how would that work under the hood? Is Aurae API expecting manifest like k8s would? Will there be "and now serialize it all to json and send" step in aurae DSL?